index.js

```diff
@@ -5,7 +5,8 @@ var once = require('once')
 var split = require('split')
 var zlib = require('zlib')
 
-var blocklistRe = /^\s*[^#].*?\s*:\s*([a-f0-9.:]+?)\s*-\s*([a-f0-9.:]+?)\s*$/
+/** this regex will math both IP ranges and single IPs, with or without a description */
+var blocklistRe = /^(?:(\s*[^#].*?)\s*:\s*)?([a-f0-9.:]+?){1}(?:\s*-\s*([a-f0-9.:]+?)\s*)?$/
```

Comment:
In the first part of the regex `(?:(\s*[^#].*?)\s*:\s*)`, there's no need for the internal capturing group `(\s*[^#].*?)`. I think we can get rid of these parens.

In the middle of the regex `([a-f0-9.:]+?){1}` there's no need for `{1}` since that just says to repeat exactly once, which is the default behavior.

At the end of the regex `(?:\s*-\s*([a-f0-9.:]+?)\s*)?` we need to ensure that the last `\s*` appears outside the optional group so that IP lines that end in trailing whitespace will still work. I'll add a test for this case.